plugins/powerd/fu-plugin-powerd.c

```diff
@@ -28,10 +28,47 @@ fu_plugin_destroy (FuPlugin *plugin)
 		g_object_unref (data->proxy);
 }
 
+static gboolean
+fu_plugin_powerd_callback(gpointer pointer)
+{
+	FuPlugin *plugin = (FuPlugin *)pointer;
+	FuPluginData *data = fu_plugin_get_data(plugin);
+	FuContext *ctx = fu_plugin_get_context(plugin);
+	guint32 power_type;
+	guint32 current_state;
+	gdouble current_level;
+	g_autoptr(GVariant) powerd_response = NULL;
+
+	/*retrieving battery info from "GetBatteryState" method call to powerd */
```

Comment:
space after the `*` -- @djcampello is that possible to encode into the clang style checks?